base_tier_validation/models/tier_validation.py

```diff
@@ -632,6 +634,18 @@ def get_view(self, view_id=None, view_type="form", **options):
                         all_models[model] = res["models"][model]
                 new_node = etree.fromstring(new_arch)
                 node.append(new_node)
+            for node in doc.xpath("//field[@name][not(ancestor::field)]"):
+                modifiers = json.loads(
+                    node.attrib.get("modifiers", '{"readonly": false}')
+                )
+                if modifiers.get("readonly") is not True:
```

Comment:
note to self: exclude the fields listed in `_get_under_validation_exceptions` here